### DISCUSSION TITLE
ploblema assenze studenti

### DISCUSSION THREAD

**Discussion by francyoo17** [iisgiua/giuaschool#712] [Richieste di aiuto]
Salve ho un ploblema con il registro quando la classe e piena di studenti e vado su assenze mi dà errore se la classe e senza studenti e vado su assenze funziona
Come posso risolvere?

**Comment by trinko** [comment]
Devi aprire una segnalazione e caricare il log generato in modalità debug.
Vedi: [Come fare una segnalazione di errore](https://github.com/iisgiua/giuaschool/discussions/576)
